<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My title</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>851</width>
     <height>331</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="rowCount">
    <number>10</number>
   </property>
   <property name="columnCount">
    <number>10</number>
   </property>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>item</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>expected pri</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>차수</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>budg</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>one pr</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>market</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>prof</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>target</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>y/n</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>period</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>auto</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>30</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>serial</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>30</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>410</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>410</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStop">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>410</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBuy">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Long</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSell">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>410</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Short</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableTarget">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>60</y>
     <width>251</width>
     <height>201</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="rowCount">
    <number>5</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Target (%)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Sell (%)</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="editProfit">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>270</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>925</x>
     <y>270</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Preserve profit (%)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <signal>signalStart()</signal>
  <slot>btnStart_Clicked()</slot>
 </slots>
</ui>
